### ISSUE TITLE
[BUG] scheduler fails after restart

### ISSUE THREAD

**Issue by zekker6** [bunkerity/bunkerweb#618]
**Description**
After the restart scheduler service is not able to start again.

**How to reproduce**

I'm using the following docker-compose configuration:
```
version: '3'
services:
  mybunker:
    image: "bunkerity/bunkerweb:1.5.1"
    restart: unless-stopped
    labels:
      - "bunkerweb.INSTANCE"
    ports:
      - 80:8080
      - 443:8443
    environment:
      - SERVER_NAME=names
      - DATABASE_URI=mariadb+pymysql://creds@bw-db:3306/db
      - MULTISITE=yes
      - BLACKLIST_RDNS=''
      - BLACKLIST_RDNS_GLOBAL=''
      - USE_DNSBL=no
      - DNS_RESOLVERS=dns_resolver_ip
      - DISABLE_DEFAULT_SERVER=yes
      - AUTO_LETS_ENCRYPT=yes
      - USE_GZIP=yes
      - USE_REVERSE_PROXY=yes
      - API_WHITELIST_IP=127.0.0.0/8 10.20.30.0/24
      - BAD_BEHAVIOR_COUNT_TIME=60
      - BAD_BEHAVIOR_THRESHOLD=5
      - BAD_BEHAVIOR_BAN_TIME=864000
      - EXTERNAL_PLUGIN_URLS=https://github.com/bunkerity/bunkerweb-plugins/archive/refs/tags/v1.1.zip
      - USE_CROWDSEC=yes
      - CROWDSEC_API=https://endpoint
      - CROWDSEC_API_KEY=key
    networks:
      - bw-services

  bw-scheduler:
    image: "bunkerity/bunkerweb-scheduler:1.5.1"
    restart: unless-stopped
    environment:
      - DATABASE_URI=mariadb+pymysql://creds@bw-db:3306/db
      - DOCKER_HOST=tcp://bw-docker:2375
    depends_on:
      - mybunker
    volumes:
      - bw-data:/data
    networks:
      - bw-services

  bw-docker:
    image: tecnativa/docker-socket-proxy
    restart: unless-stopped
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
    environment:
      - CONTAINERS=1
      - LOG_LEVEL=warning
    networks:
      - bw-services

  bw-db:
    image: mariadb:10.10
    restart: unless-stopped
    volumes:
      - db-data:/var/lib/mysql
    networks:
      - bw-services

volumes:
  bw-data:
  db-data:

networks:
  bw-services:
    name: bw-services
    ipam:
      driver: default
      config:
        - subnet: 10.20.30.0/24
```

After the restart scheduler service can't be started again.

This started happening after I've removed crowdsec related options to test if it is related to some other issue.
Also tried removing all crowdsec options and `EXTERNAL_PLUGIN_URLS`, but that didn't help.

**Logs**

```
Sep  5 16:43:39 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:39] - GENERATOR - ℹ️  - Save config started ...
Sep  5 16:43:39 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:39] - GENERATOR - ℹ️  - Settings : /usr/share/bunkerweb/settings.json
Sep  5 16:43:39 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:39] - GENERATOR - ℹ️  - Core : /usr/share/bunkerweb/core
Sep  5 16:43:39 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:39] - GENERATOR - ℹ️  - Plugins : /etc/bunkerweb/plugins
Sep  5 16:43:39 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:39] - GENERATOR - ℹ️  - Init : False
Sep  5 16:43:39 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:39] - GENERATOR - ℹ️  - Checking arguments ...
Sep  5 16:43:39 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:39] - GENERATOR - ℹ️  - Database connection established
Sep  5 16:43:39 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:39] - GENERATOR - ℹ️  - Computing config ...
Sep  5 16:43:40 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:40] - GENERATOR - ⚠️  - Ignoring variable BLACKLIST_RDNS_GLOBAL : value '' doesn't match regex ^(yes|no)$
Sep  5 16:43:40 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:40] - GENERATOR - ℹ️  - Database is already initialized, skipping ...
Sep  5 16:43:44 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:44] - GENERATOR - ℹ️  - Config successfully saved to database
Sep  5 16:43:44 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:44] - GENERATOR - ℹ️  - Config saver successfully executed !
Sep  5 16:43:46 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:46] - SCHEDULER - ℹ️  - Database connection established
Sep  5 16:43:47 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:47] - SCHEDULER - ℹ️  - Scheduler started ...
Sep  5 16:43:47 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: [2023-09-05 16:43:47] - SCHEDULER - ❌ - Exception while executing scheduler : Traceback (most recent call last):
Sep  5 16:43:47 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]:   File "/usr/share/bunkerweb/scheduler/main.py", line 433, in <module>
Sep  5 16:43:47 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]:     err = db.update_external_plugins(external_plugins, delete_missing=True)
Sep  5 16:43:47 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]:           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Sep  5 16:43:47 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]:   File "/usr/share/bunkerweb/db/Database.py", line 1138, in update_external_plugins
Sep  5 16:43:47 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]:     updates[Plugins.data] = plugin["data"]
Sep  5 16:43:47 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]:                             ~~~~~~^^^^^^^^
Sep  5 16:43:47 bunkerweb_bw-scheduler_1/5e657c29faa5[661306]: KeyError: 'data'
```

**Comment by TheophileDiot** [comment]
Hi @zekker6, thank you for opening this issue, we are aware of this bug with the 1.5.1 version and external plugins. It will be fixed in the next version (1.5.2)

**Comment by zekker6** [comment]
@TheophileDiot Thank you for letting me know! 
Any chance there is a workaround available before 1.5.2 release?

**Comment by fl0ppy-d1sk** [comment]
Hello @zekker6,

It should be fixed in the last [1.5.2 version](https://github.com/bunkerity/bunkerweb/releases/tag/v1.5.2). Feel free to open a new issue if needed.
